src/test/java/core/basesyntax/services/impl/CsvFileWriterImplTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.junit.Test;
+
+public class CsvFileWriterImplTest {
+
+    @Test
+    public void createReportFileMethodTest_Ok() {
+        List<String> expectedList = List.of("fruit,quantity",
+                "How are u doing",
+                "wass up","100%N....");
+        List<String> data = List.of("How are u doing","wass up","100%N....");
+        new CsvFileWriterImpl().createReportFile("src/test/testSource/reportTest.csv", data);
+        List<String> actualList = new CsvFileReaderImpl()
+                .getAcceptedFileAsList("src/test/testSource/reportTest.csv");
```

Comment:
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/58032907/200301894-ad379498-cd91-40d0-bc6a-46d040a2ca16.png">

Reply:
can u suggest me what i should do/ i cant understand how to create this test without reader

Reply:
you can use `Files.readLines`, for example